pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[tool.poetry]
+name = "manimlib"
+version = "0.2.0"
+description = "Animation engine for explanatory math videos."
+authors = ["3b1b <grant@3blue1brown.com>"]
+license="MIT"
+readme="README.md"
+repository="https://github.com/manimcommunity/manim"
+documentation="https://manimce.readthedocs.io/en/latest/"
+classifiers= [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: MIT License",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Multimedia :: Video",
+    "Topic :: Multimedia :: Graphics",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Natural Language :: English",
+    ]
+exclude = ["scripts/","logo/","readme-assets/"]
+packages = [
+    { include = "manim" },
+]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+colour = "*"
+numpy = "*"
+Pillow = "*"
+progressbar = "*"
+scipy = "*"
+tqdm = "*"
+pydub = "*"
+pygments = "*"
+pyreadline = { version = "*", markers = "sys_platform == 'win32'" }
+rich = ">=4.2.1"
+pycairo = [
+	{ version = "^1.19", markers = "sys_platform != 'win32'"},
+	{ version = "^1.19", markers = "sys_platform == 'win32'", source="testpypi" },
+]
+dataclasses = {version = "^0.7", python = "~3.6"}
+
+[tool.poetry.dev-dependencies]
+pytest = "^6.0"
+
+[tool.poetry.dev-dependencies.black]
+version = "^20.8b1"
+allow-prereleases = true
+python = "^3.6"
+markers = "platform_python_implementation == 'CPython'"
+
+[[tool.poetry.source]]
+name = "testpypi"
+url = "https://test.pypi.org/simple/"
+secondary = true
+
+[tool.pytest.ini_options]
+markers = "slow: Mark the test as slow. Can be skipped with --skip_slow"
```

Comment:
@huguesdevimeux I have changed `pytest.ini` into `pyproject.toml`. Could you check whether this work?  Docs [here](https://docs.pytest.org/en/stable/customize.html).

Reply:
Just checked: this works.